Library of Congress ISBN.js

```diff
@@ -8,7 +8,7 @@
 	"priority": 98,
 	"inRepository": true,
 	"translatorType": 8,
-	"browserSupport": "gcsb",
+	"browserSupport": "gcsibv",
```

Comment:
Why don't we need to bump the timestamp here? E.g. when using the bookmarklet to save to Standalone this would matter, no?